docs/docusaurus.config.js

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2024 Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e.V.
+ *
+ * This work is licensed under the Fraunhofer License (on the basis of the MIT license)
+ * that can be found in the LICENSE file.
+ *
+ * The initial version of this file was auto-generated by Docusaurus.
+ */
+
+// @ts-check
```

Comment:
A lot of commented out code snippets in this file. Is there a purpose for it?

Reply:
Lines like `// @ts-check` seem to be commented out but have a special purpose. See https://www.typescriptlang.org/docs/handbook/intro-to-js-ts.html for more details.